feature-libs/product-configurator/rulebased/root/rulebased-configurator-root-feature.module.ts

```diff
@@ -3,7 +3,7 @@ import { provideDefaultConfig } from '@spartacus/core';
 
 const cmsComponents: string[] = [
   'ConfiguratorForm',
-  'ConfiguratorOverview',
+  'ConfiguratorOverviewForm',
```

Comment:
is it breaking change?

Reply:
picked

Reply:
No! This is a bug fix. There is no 'ConfiguratorOverview' component, so it was a typo.
(Which doesn't have any real effect here, as we list the anchor components that trigger our feature. And whenever the overview form is active, other components are present which trigger our feature as well)